src/main/java/com/ldtteam/structurize/client/gui/WindowScan.java

```diff
@@ -344,6 +344,7 @@ private void updateResources()
                     for (final Entity entity : list)
                     {
                         if (!entities.containsKey(entity.getName().getString())
+                                && entity.getType().canSerialize()
```

Comment:
While I generally agree with this, can you exclude EntityType.LEASH_KNOT from this check? lighting bolt, player and fishing bobber are fine, but we want to list leash cuz currently we can't place them nor render them (in preview) properly - so builders should have way to check & remove them easily

Reply:
All the no-can-serialize entities are simply not saved at actual scan time, which means that they're effectively removed at that point anyway.  If there's a knot-to-knot leash then both ends go away; if there's a leash-to-animal then the animal might remain and be confused at the other end, but they'd appear in the list and that should be suspicious enough because animals shouldn't be in the scan either.

So I'm not sure why this isn't better as is -- it should just work without having to remember to do anything.  Can you explain further what the issue is?

Reply:
Well, leash is normal entity that just has bullshit serialization, it's not unserializable entity in same way like bolt/bobber - it's serialized on the owning entity, thus if you place the entity with knot, that wasn't updated, then the know just drops as item

Reply:
Ok, but if someone ties an animal to a post and scans it, then builds that scan and the animal appears and a lead drops from them -- that's less ideal than if the lead were tied properly, but it still seems sufficiently correct; everything that was in the scan ended up back in the world, including the lead.  And if the player ignores it for five minutes then it ends up in the state you seem to be suggesting, where everything else is the same but the lead is gone.

Still, given that it does drop a lead item, you're right that it's probably worth explicitly alerting the player that there is a knot in the scan area, even if they don't remove it.  Having said that, it probably wouldn't be hard to make them actually work.  But that'd be a separate PR.

Reply:
Making them work across whole structurize is bit pain, was looking into it few months ago. Need to look at how does vanilla templates solve that, if they don't file bug report to mojang and wait for them to make fix first (since it's not something game-breaking)